src/scripts/settings.js

```diff
@@ -31,6 +41,20 @@ export const imageTitles = [
     'Runes of Prosperity by Rin'
 ];
 
+export const deckImageTitles = [
```

Comment:
Temporary for now. I shifted everything by 1 index so on deck will use a different marker. Thinking of making a custom marker for this, maybe some sort of rectangle icon?